.github/workflows/stale.yml

```diff
@@ -0,0 +1,26 @@
+name: Check stale issues and pull requests
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 12 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v5
```

Comment:
Is there a way to exempt untriaged issues? I don't think we want to autoclose issues we never even looked at.

Reply:
We can pass the option `any-of-labels` with the labels that we want to consider. So unlabeled issues/PR will never be checked.

Reply:
I guess might be good enough. We'd just need a rule not to partially label the issues. All or none.

Reply:
Let's do this.